src/SFA.DAS.FAA.Web/Views/Apply/Index.cshtml

```diff
@@ -164,7 +164,7 @@
             </li>
         </ul>
 
-        @if (Model.IsDisabilityConfident)
+        @if (true)
```

Comment:
This might have been done in error I think

Reply:
yes, it was a wrong commit and corrected that in the subsequent commit,